src/test/regress/expected/multi_view.out

```diff
@@ -251,6 +251,75 @@ SELECT * FROM mode_counts WHERE l_shipmode = 'AIR' ORDER BY 2 DESC, 1 LIMIT 10;
 (1 row)
 
 DROP MATERIALIZED VIEW mode_counts;
+-- modify statements on/with views
+-- create two tables and a view
+CREATE TABLE large (id int, tenant_id int);
+CREATE TABLE small (id int, tenant_id int);
+SELECT create_distributed_table('large','tenant_id');
+ create_distributed_table 
+--------------------------
+ 
+(1 row)
+
+SELECT create_distributed_table('small','tenant_id');
+ create_distributed_table 
+--------------------------
+ 
+(1 row)
+
+CREATE VIEW small_view AS SELECT * from small where id < 100;
```

Comment:
Can you also add similar tests with materialized views?

Reply:
I will add them asap